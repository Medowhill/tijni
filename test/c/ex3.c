void f(long ptr, int x) {
  int *p = (int *) ptr;
  *p = x;
}
