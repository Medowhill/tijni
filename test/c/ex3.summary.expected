{
  "f": {
    "params": [ { "name": "%ptr(f)", "used": true, "freed": false }, { "name": "%x(f)", "used": false, "freed": false } ],
    "ret": [],
    "env": {
      "%ptr(f)": [ "'%ptr(f)" ],
      "%x(f)": [ "'%x(f)" ],
      "%0(f)": [ "'%ptr(f)" ],
      "'%ptr(f)": [ "'%x(f)" ]
    }
  }
}